engine/context/ecs-integration.md

```diff
@@ -119,7 +176,7 @@ If required, the initial file system can be customized using `driver-opts`:
 
 ```yaml
 volumes:
-  my-data: 
+  my-data:

```

Comment:
Is this the simplest Compose file to create a new volume or are some of these optional?

Reply:
all optionals. Can write:
```yaml
volumes:
  my-data: {}
```
Would then be created with EFS defaults

Reply:
I added a sample compose service with the most basic volume definition possible, useful indeed